solidity/contracts/converter/types/standard-pool/StandardPoolConverter.sol

```diff
@@ -986,6 +1080,23 @@ contract StandardPoolConverter is
         return reserveAmounts;
     }
 
+    function reserveBalancesMinusFees(IERC20[] memory _reserveTokens) internal view returns (uint256[2] memory) {
```

Comment:
fees need to be calculated based on the fee calculation formula, not by subtracting the previous balance from the current balance.
consider a case in which you have previous fee balances of 100,100 and after a swap, the balances are 200,60 - what would the fee be in this case?